moveit_ros/visualization/motion_planning_rviz_plugin/include/moveit/motion_planning_rviz_plugin/motion_planning_display.h

```diff
@@ -55,6 +56,7 @@
 #endif
 
 #include <moveit_msgs/DisplayTrajectory.h>
+#include <QDockWidget>
```

Comment:
I believe this is not needed, because now you use `rviz::PanelDockWidget` instead, right?
Could you please remove it from the request?